<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compilador</class>
 <widget class="QWidget" name="compilador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compilador</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QTextEdit" name="textEditar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textErrores">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>781</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="botonCompilar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Compilar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonCancelar">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botonCancelar</sender>
   <signal>clicked()</signal>
   <receiver>compilador</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
